sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -119,10 +119,10 @@ def test_valid_alpha():
     X, y = make_classification(n_classes=n_classes, n_samples=200,
                                random_state=0)
     for alpha in [-0.1, 0, 1, 1.1, None]:
-        assert_raises(ValueError,
-                      lambda **kwargs:
-                      label_propagation.LabelSpreading(**kwargs).fit(X, y),
-                      alpha=alpha)
+        with pytest.raises(ValueError):
+            (lambda **kwargs:
```

Comment:
That is a weird use case for lambda. This should also work:

```py
label_propagation.LabelSpreading(alpha=alpha).fit(X, y)
```

Reply:
@thomasjpfan I have corrected it. Let me know if anything else needs to be done.